control-plane/agents/core/src/server.rs

```diff
@@ -75,6 +76,11 @@ pub(crate) struct CliArgs {
     /// Trace rest requests to the Jaeger endpoint agent
     #[structopt(long, short)]
     jaeger: Option<String>,
+    /// The GRPC Server URLs to connect to
+    /// (supports the http/https schema)
+    /// Default: 0.0.0.0:50051
```

Comment:
probably don't need this line, structopt will already show the default by itself